prettier/main.ts

```diff
@@ -227,6 +228,64 @@ async function formatSourceFiles(
   exit(0);
 }
 
+function filesSelector(
```

Comment:
`filesSelector` as function sounds a little strange to me. How about something like `getFilesToFormat` or `getTargetFiles`?

And this function looks a little complex. I think it's better to add some `jsdoc` before this line.